libtiledbsoma/src/CMakeLists.txt

```diff
@@ -170,6 +171,7 @@ endif()
 install(FILES 
   ${CMAKE_CURRENT_SOURCE_DIR}/soma/managed_query.h
   ${CMAKE_CURRENT_SOURCE_DIR}/soma/soma_array.h 
+  ${CMAKE_CURRENT_SOURCE_DIR}/soma/soma_group.h 
```

Comment:
There is no `soma_group.h` in the PR as best as I can see. Forgot a `git add` ?

Reply:
Yeah I was working on two branches at once and accidentally commit SOMA Group work to this PR.